ansible/roles/ocp4-workload-migration/defaults/main.yml

```diff
@@ -12,7 +12,15 @@ mig_expected_crds:
 mig_state: "present"
 mig_migration_namespace: "openshift-migration"
 mig_download_content:
-- "https://raw.githubusercontent.com/konveyor/mig-agnosticd/master/demos/2020_Summit/labs/scripts/bookbag.yml"
+- url: "https://raw.githubusercontent.com/konveyor/mig-agnosticd/master/demos/2020_Summit/labs/scripts/bookbag.yml"
```

Comment:
I guess this is better since it's pointing to 2020_summit. But would a tag be even better?